staging/src/k8s.io/apiserver/pkg/server/egressselector/config_test.go

```diff
@@ -511,30 +511,6 @@ func TestValidateEgressSelectorConfiguration(t *testing.T) {
 				},
 			},
 		},
-		{
-			name:        "both master and controlplane egress selection configured",
```

Comment:
add a unit test for duplicate items in the list

Reply:
Added.